### PR TITLE
process_status: clean up the docs

### PR DESCRIPTION
**Pull request by lasers**
Seperate pull request for `process_status`. We make the docs suck less... Should I start merging formats everywhere? EDIT: This is the last `default format for this module` string. 👌

### CODE REVIEW

**Comment by tobes** [comment]
Cool.  I'll merge when travis is our friend again

File: py3status/modules/process_status.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 10)
-    format: default format for this module (default '{icon}')
-    full: if True, match against full command line (default False)
-    icon_off: show when process not running (default '■')
-    icon_on: show when process running (default '●')
-    process: process name to check for (default None)
+    format: display format for this module (default '{icon}')
+    full: match against the full command line (default False)
+    icon_off: show icon if process is not running (default '■')
```
Comment (tobes) [inline py3status/modules/process_status.py]: show __as__ icon ...

Comment (lasers) [reply py3status/modules/process_status.py]: That does not make sense (to me) as this is the only option to show (as). Anyhow, I pushed out something a bit different... and we can  `mergeformat` this soon or later to get rid of the descriptions / lines / nitpicks. Proceed with `mergeformat`? .__.

**Comment by tobes** [comment]
Thanks @lasers better documentation is always good

**Pull request merged**